src/vm/runners/builtin_runner/bitwise.rs

```diff
@@ -115,6 +119,32 @@ impl BitwiseBuiltinRunner {
         ("bitwise", (self.base, self.stop_ptr))
     }
 
+    pub fn get_used_cells_and_allocated_size(
+        self,
+        vm: &VirtualMachine,
```

Comment:
Can we receive `vm.current_step` instead of the whole `vm`?

Reply:
We cant at least with this implementation due to the fact that we need to retrieve the used cells with the get_used_cells